scripts/peer-api-check.js

```diff
@@ -15,7 +15,6 @@
  */
 
 const path = require('path');
-const semver = require('semver');
```

Comment:
can also be removed from 'scripts/package.json`

Reply:
done